doc/content/examples/example9.md

```diff
@@ -0,0 +1,82 @@
+
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Acceleration response spectra of the soil subjected to the Ormsby wavelet.
+
+### Soil-structure interaction analysis
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra at 10m height from the free-field soil is computed. Free-field soil is the soil region unaffected by the structural response, which, in this case, is a corner node on the surface of the soil domain. [fig:SSI_Amp] presents the amplification ratio as a function of the period.
+
+!listing examples/ex09/SSI_Response.i
+
+!media media/examples/ex09_SSI.png
+       style=width:80%;margin-right:0px;float:center;
+       id=fig:SSI_Amp
+       caption=Amplification ratio of the structural response spectra from the free-field soil in SSI analysis. This is computed using MASTODON.
+
+[fig:SSI_FreeField] presents the free-field soil acceleration response (1% damped) from the SSI analysis, compared to that obtained from the site response analysis. It is seen that these to spectra are very similar. The subtle differences are likely due to the coarse mesh size and small domain extent for the soil-structure interaction problem.
+
+!media media/examples/ex09_SSI_FreeField.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:SSI_FreeField
+       caption=Free-field response spectra from the SSI analysis and the individual soil analysis.
+
+For this example, an INL supercomputer is used with dt of 0.002 sec for a total of 1500 time steps due to its expensive computational cost. It is not recommended to run this problem on your local machine. For details on how to use an INL supercomputer, please see [FAQ](help/faq.md). On FALCON machine with 1 full node, this problem takes about 3-3.5 hours to complete.
```

Comment:
Remove this paragraph. We don't need to provide guidance on what machine to use.